src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,35 +50,42 @@ public String todayDate(DateTimePart datePart) {
      * Return Optional of a date built from these elements.
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        if (dateParams.length == 0) {
+            return Optional.empty();
+        }
```

Comment:
alternatively, we may catch ArrayIndexOutOfBoundsException along with DateTimeException